### PR TITLE
Update registration-auth.md

### PR DESCRIPTION
**Pull request by JuanitaBaptiste**
updated to Microsoft admin center

### CODE REVIEW

File: memdocs/autopilot/registration-auth.md [hunk 1]
```diff
@@ -82,7 +82,8 @@ For a CSP to register Windows Autopilot devices for a customer, the customer mus
 Each OEM has a unique link to provide to their respective customers, which the OEM can request from Microsoft via msoemops@microsoft.com.
 
 1. OEM emails link to their customer.
-2. Customer with Microsoft Store for Business (MSfB) global administrator privileges clicks the link in the email, which takes them directly to the following MSfB page:
+2. Customer logs into the Microsoft Admin Center using a cloud native account (example: [domain].onmicrosoft.com) with global administrator privileges.
```
Comment (JohanFreelancer9) [inline memdocs/autopilot/registration-auth.md:85]:
```suggestion
2. Customer logs into Microsoft admin center using a cloud-native account (for example, [domain].onmicrosoft.com) with Global administrator privileges.
```

File: memdocs/autopilot/registration-auth.md [hunk 2]
```diff
@@ -82,7 +82,8 @@ For a CSP to register Windows Autopilot devices for a customer, the customer mus
...
+3. Clicks the link in the email, which takes them directly to the following page:
```
Comment (JohanFreelancer9) [inline memdocs/autopilot/registration-auth.md:86]:
```suggestion
3. Selects the link in the email, which takes them directly to the following page:
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @JuanitaBaptiste, kindly commit the suggested changes. Many thanks.

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
@JuanitaBaptiste, you seem to have missed one suggested change. Could you also commit that, please?

**Review by JohanFreelancer9 — APPROVED** [review]

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by frankroj** [comment]
Reached out to contributor to clarify which admin center they are referring to in the change. Also inquired if a change in screen shots may be necessary.

**Comment by traya1** [comment]
Learn Build status updates of commit _[2cd10c2](https://github.com/JuanitaBaptiste/memdocs/commits/2cd10c2daec440d8d474a718b434f7955d461c03)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[memdocs/autopilot/registration-auth.md](https://github.com/JuanitaBaptiste/memdocs/blob/patch-6/memdocs/autopilot/registration-auth.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://buildapi.docs.microsoft.com/Output/PullRequest/aa268e4d-aaaf-30c8-4118-35d62ca794c9/202212272105201992-3391/BuildReport?accessString=323796b416de04485df12a3a338a7d4ca8019e9d0814b814f9d74ab2334a2d77).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/aa268e4d-aaaf-30c8-4118-35d62ca794c9?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching the learn.microsoft.com <a href="https://review.learn.microsoft.com/en-us/help/?branch=main">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Learn support channel</a></li></ul>

**Comment by frankroj** [comment]
Approving changes. Need to update graphics but I will do that in a separate PR

**Comment by frankroj** [comment]
#sign-off

**Pull request merged**